lib/elixir_boilerplate_graphql/elixir_boilerplate_graphql.ex

```diff
@@ -19,6 +19,11 @@ defmodule ElixirBoilerplateGraphQL do
 
     config
     |> Absinthe.Plug.default_pipeline(options)
+    |> Pipeline.insert_after(Absinthe.Phase.Document.Complexity.Result, {AbsintheSecurity.Phase.IntrospectionCheck, options})
+    |> Pipeline.insert_after(Absinthe.Phase.Document.Result, {AbsintheSecurity.Phase.FieldSuggestionsCheck, options})
+    |> Pipeline.insert_after(Absinthe.Phase.Document.Complexity.Result, {AbsintheSecurity.Phase.MaxAliasesCheck, options})
+    |> Pipeline.insert_after(Absinthe.Phase.Document.Complexity.Result, {AbsintheSecurity.Phase.MaxDepthCheck, options})
+    |> Pipeline.insert_after(Absinthe.Phase.Document.Complexity.Result, {AbsintheSecurity.Phase.MaxDirectivesCheck, options})
     |> Pipeline.insert_before(Result, Middleware.OperationNameLogger)
     |> Pipeline.insert_after(Result, Middleware.ErrorReporting)
```

Comment:
@remi here 😃 

We might want to cleanup the aliases at the same time…

```suggestion
    |> Pipeline.insert_after(Absinthe.Phase.Document.Complexity.Result, {AbsintheSecurity.Phase.IntrospectionCheck, options})
    |> Pipeline.insert_after(Absinthe.Phase.Document.Complexity.Result, {AbsintheSecurity.Phase.MaxAliasesCheck, options})
    |> Pipeline.insert_after(Absinthe.Phase.Document.Complexity.Result, {AbsintheSecurity.Phase.MaxDepthCheck, options})
    |> Pipeline.insert_after(Absinthe.Phase.Document.Complexity.Result, {AbsintheSecurity.Phase.MaxDirectivesCheck, options})
    |> Pipeline.insert_before(Absinthe.Phase.Document.Result, Middleware.OperationNameLogger)
    |> Pipeline.insert_after(Absinthe.Phase.Document.Result, {AbsintheSecurity.Phase.FieldSuggestionsCheck, options})
    |> Pipeline.insert_after(Absinthe.Phase.Document.Result, Middleware.ErrorReporting)
```